backend/build.gradle

```diff
@@ -60,6 +60,9 @@ dependencies {
     // Jsoup for input sanitization
     implementation 'org.jsoup:jsoup:1.13.1'
 
+    // graphql-java-extended-validation schema directives
+    compile 'com.graphql-java:graphql-java-extended-validation:16.0.0'
```

Comment:
Think this is the documentation? https://github.com/graphql-java/graphql-java-extended-validation

It supports richer "EL" syntaxes, too like `@Digits`,  `@Expression`, `@Range`.

Maybe reference this documentation/features somewhere? Not sure where though... also maybe it's not really that important.

Reply:
I added a link and some words to the GraphQL schema doc. Not sure how discoverable that is, but hopefully anybody looking to add directives will go through that file.